src/main/resources/report-header.html

```diff
@@ -0,0 +1,56 @@
+<html>
+    <head>
+        <title>Deprecated API Usage in Plugins</title>
+        <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
+        <style type="text/css">
+* {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+}
+body > h1, body > h2, body > h3 {
+    text-align: center;
+}
+h1 {
+    margin-top: 30px;
```

Comment:
Margins on top/bottom are for visual separation, so less of a big deal.

Also, 2x WQHD, so not quite 5k. Also, I generally zoom in quite a bit, I'm not 16 anymore.